src/application.coffee

```diff
@@ -604,14 +604,19 @@ application.update = update = (force, scheduled = false) ->
 		when UPDATE_SCHEDULED
 			if scheduled isnt true
 				# There's an update scheduled but it isn't this one, so just stop
+				# but if we have to force an update, do it in the one that is scheduled
+				updateStatus.forceNext = updateStatus.forceNext or force
 				return
 		when UPDATE_IDLE
 			# All good, carry on with the update.
 		else
 			# Mark an update required after the current in-progress update.
-			updateStatus.forceNext = force
+			updateStatus.forceNext = updateStatus.forceNext or force
 			updateStatus.state = UPDATE_REQUIRED
 			return
+	if scheduled is true
```

Comment:
Why this check? Do we care if it's scheduled or not for the sake of `forceNext`?